.github/workflows/tests.yaml

```diff
@@ -67,3 +67,34 @@ jobs:
           echo ${{ secrets.GITHUB_TOKEN }} | docker login $IMAGE_REGISTRY -u $GITHUB_ACTOR --password-stdin
           docker pull $WEB_IMAGE
           docker build --cache-from $WEB_IMAGE --target development --tag web-dev ./web
+
+  test-docs:
+    name: test-docs
+    runs-on: ubuntu-latest
+
+    steps:
+      # If you know your docs does not rely on anything outside of the documentation folder, the commented out code below can be used to only test the docs build if the documentation folder changes.
+      - name: "Checkout GitHub Action"
+        uses: actions/checkout@main
+        # with:
+        #   fetch-depth: 0
+
+      # - name: "Get number of changed documentation files"

```

Comment:
There are some commented-out things, should it be removed?

Reply:
I have a comment above explaining why they are commented out